<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>601</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bitset:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ans">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>511</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="allButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="anyButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Any</string>
    </property>
   </widget>
   <widget class="QPushButton" name="countButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Count</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_reset">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_resetArg">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_flip">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QPushButton" name="noneButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_flipArg">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Flip</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ullongButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>To_ullong</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_testArg">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_set">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>180</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QPushButton" name="getSizeButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>getSize</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_test">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>260</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>9</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_setArg">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ulongButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>To_ulong</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="flipButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Flip</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
